src/SpectatorView.Unity/Assets/SpectatorView.Editor/Scripts/CompositorWindowBase.cs

```diff
@@ -19,6 +19,18 @@ internal class CompositorWindowBase<TWindow> : EditorWindowBase<TWindow> where T
         protected const string AppDeviceTypeLabel = "App";
         protected string holographicCameraIPAddress;
 
+        /// <summary>
+        /// This list contains localizers that can't be used in the compositor window for video camera filming.
+        /// </summary>
+        private readonly List<Guid> unsupportedLocalizers = new List<Guid>

```

Comment:
Would it potentially be better to have a list of "compositorSupported" localizers, and then list the ones that are supported? It seems to me like any new future localizers are likely to not be supported by default unless something is done to them to make them work.

Reply:
I think I would prefer to have an exclusion list. compositorSupported localizers would require us to add some mechanism for registering localizers as supported for external contributors. Or they would have to edit this source which may be distributed through nuget as compiled binaries.